app/models/visit_group.rb

```diff
@@ -80,6 +81,10 @@ def in_order?
     arm.visit_groups.where("position < ? AND day >= ? OR position > ? AND day <= ?", position, day, position, day).none?
   end
 
```

Comment:
This calculation was originally done on each visit input to include a `cents` value which was passed to the javascript totals calculation. Instead, we used this method to get the totals in the new partials.